modules/tinymce/Gruntfile.js

```diff
@@ -72,18 +74,16 @@ module.exports = function (grunt) {
       {
         core: {
           options: {
-            treeshake: true,
```

Comment:
I realise this is removed because it's the default, but did you consider any of the tree shaking options rollup now offers? `propertyReadSideEffects` could be good for us 🤔

Reply:
I think that is enabled by default. I haven't tinkered that much with the options since I focused on just getting this to work. Was more work than I thought. 😓